apis/infix-en_GB/atrium-api-infix-en_GB-common/src/test/kotlin/ch/tutteli/atrium/api/infix/en_GB/samples/ListAssertionSamples.kt

```diff
@@ -0,0 +1,57 @@
+package ch.tutteli.atrium.api.infix.en_GB.samples
+
+import ch.tutteli.atrium.api.infix.en_GB.*
+import ch.tutteli.atrium.api.verbs.internal.expect
+import ch.tutteli.atrium.translations.DescriptionComparableAssertion
+import kotlin.test.Test
+
+class ListAssertionSamples {
+    private val isLessThanDescr = DescriptionComparableAssertion.IS_LESS_THAN.getDefault()
+    private val isGreaterThanDescr = DescriptionComparableAssertion.IS_GREATER_THAN.getDefault()
+
+    @Test
+    fun getFeature() {
+        val list = listOf(1, 2, 3)
+
+        expect(list) get 0 isLessThan 2
+        expect(list) get 0 isGreaterThan 0
+
+        fails {
+            expect(list) get 3 isLessThan 0
```

Comment:
```suggestion
            expect(list) get 3 isLessThan 0
            //                |       |  not reported
            //                | fails because index 3 is out of bound
            // use `get(index) { ... }` if you want that all assertions are evaluated                              

```